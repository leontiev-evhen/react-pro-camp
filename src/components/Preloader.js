import React from 'react';
import { Spinner } from 'react-bootstrap';

export function Preloader() {
	return (
		<div className="preloder">
			<Spinner animation="grow" variant="danger" className="loader" />
		</div>
	);
}
