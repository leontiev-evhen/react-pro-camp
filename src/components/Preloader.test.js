import React from 'react';
import { mount } from 'enzyme';
import { Preloader } from './Preloader';

describe('Preloader component', () => {
	const Component = mount(<Preloader />);
	it('Preloader component render correctly', () => {
		expect(Component).toMatchSnapshot();
	});
});
