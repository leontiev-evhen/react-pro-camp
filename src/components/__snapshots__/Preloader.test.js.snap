// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader component Preloader component render correctly 1`] = `
<Preloader>
  <div
    className="preloder"
  >
    <Bootstrap(Spinner)
      animation="grow"
      className="loader"
      variant="danger"
    >
      <Spinner
        animation="grow"
        bsPrefix="spinner"
        className="loader"
        variant="danger"
      >
        <div
          className="loader spinner-grow text-danger"
        />
      </Spinner>
    </Bootstrap(Spinner)>
  </div>
</Preloader>
`;
